js/apps/filter-page/directives/tag.js

```diff
@@ -3,11 +3,14 @@
 
 	const tagDirective = (tagParsingService) => {
 		const tagLink = function filterLink($scope) {
-			$scope.toggleFilter = (activeFilter, $event) => {
-				const $element = angular.element($event.currentTarget);
+			$scope.toggleFilter = (activeFilter) => {
+				const tagFamiliesForCard = tagParsingService.parseTags($scope.tagData.attributes);
+				const activeTagName = tagParsingService.extractTagName(activeFilter);
+				const activeTagFamily = tagFamiliesForCard.filter((tagFamily) => {
+					return tagFamily.tags.indexOf(activeTagName) !== -1;
+				})[0];
```

Comment:
Will this array ever be null?

Reply:
I don't think it's possible, but I added code to protect against it if it ever happens.